readme.md

```diff
@@ -31,6 +31,32 @@ To use the custom embed you will need to:
 <lite-youtube videoid="ogfYd705cRs"></lite-youtube>
 ```
 
+### Additional Parameters
+
+YouTube supports a variety of different [player parameters](https://developers.google.com/youtube/player_parameters).
+These may be applied by using the `params` property and attribute.
+The `autoplay` attribute defaults to `1` for the lite embed element.
+
+```html
+<!-- Example to show a video player without controls -->
+<lite-youtube videoid="ogfYd705cRs" params="&controls=0"></lite-youtube>
+```
+
+```javascript
+// Create your element or get it via a DOM reference
+const embeddedPlayer = new LiteYTEmbed();
```

Comment:
is this an important usecase?
seems like we add some extra code in this PR to do property->attribute reflection that i'd ideally like to not do, but i also dont wanna fight if this JS instantiation pattern is common in the WC world.

Reply:
It is an important use-case. From Material Design Lite -> Material Components for Web experience, dynamic JS creation is very much what some set of developers desire. It's not difficult to support so long as you consider it when designing.